numpy/core/src/private/ufunc_override.c

```diff
@@ -49,11 +49,26 @@ has_non_default_array_ufunc(PyObject *obj)
 }
 
 /*
+ * Check whether an object sets __array_ufunc__ = None.
+ */
```

Comment:
Might note that the existence of the attribute must be checked before calling this.